src/core/vectortile/qgsvectortilewriter.h

```diff
@@ -128,7 +130,12 @@ class CORE_EXPORT QgsVectorTileWriter
     void setDestinationUri( const QString &uri ) { mDestinationUri = uri; }
 
     /**
-     * Sets extent of vector tile output. Currently always in EPSG:3857.
+     * Sets zoom level 0 tile matrix
+     */
+    void setZ0TileMatrix( const QgsTileMatrix &tileMatrix ) { mZ0TileMatrix = tileMatrix; }
```

Comment:
Can we rename this to `setRootTileMatrix` ? the current name looks a bit cryptic :)

Reply:
Thanks for the idea @wonder-sk I have renamed this method and private attribute.